src/antsibull/cli/antsibull_lint.py

```diff
@@ -8,150 +8,20 @@
 Entrypoint to the antsibull-changelog script.
 """
 
-import argparse
-import os.path
 import sys
-import traceback
 
-from typing import Any, List
+from typing import List
 
-try:
-    import argcomplete
-    HAS_ARGCOMPLETE = True
-except ImportError:
-    HAS_ARGCOMPLETE = False
 
-from antsibull_changelog.lint import lint_changelog_yaml
-from antsibull_changelog.logger import setup_logger
-
-from antsibull_core.args import get_toplevel_parser, normalize_toplevel_options
-
-from antsibull_docs.lint_extra_docs import lint_collection_extra_docs_files
-from antsibull_docs.collection_links import lint_collection_links
-
-
-def run(args: List[str]) -> int:
+def run(args: List[str]) -> int:  # pylint: disable=unused-argument
     """
     Main program entry point.
     """
```

Comment:
This is nicer than disabling the check, IMO.

```suggestion
def run(args: List[str]) -> int:
    """
    Main program entry point.
    """
    del args
```

Reply:
I guess I like my approach better, because it requires one less ugly noqa comment, it's shorter, and makes more sense in my head :shrug:.

Reply:
I'm still torn. I think I'll keep it as-is for now, but I'm not done pondering this question... ;)